test/Dapr.Actors.Test/Dapr.Actors.Test.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net5</TargetFrameworks>
```

Comment:
FYI what this means is that the tests will build and test separately with different versions of the runtime. The shipping code is unchanged - it compiles against 3.1 and will run with whatever version the application is using.